src/realm-settings/KeysProvidersTab.tsx

```diff
@@ -75,11 +78,11 @@ export const KeysTabInner = ({ components, refresh }: KeysTabInnerProps) => {
   const [isCreateModalOpen, setIsCreateModalOpen] = useState(false);
 
   const serverInfo = useServerInfo();
-  const providerTypes = serverInfo.componentTypes![
-    "org.keycloak.keys.KeyProvider"
-  ].map((item) => item.id);
+  const providerTypes = serverInfo.componentTypes![KeyProviderType].map(
+    (item) => item.id
+  );
```

Comment:
It might be better to use optional chaining with a default here in case the value is not set.
```suggestion
  const providerTypes = (
    serverInfo.componentTypes?.[KeyProviderType] ?? []
  ).map((item) => item.id);
```